.gitignore

```diff
@@ -76,4 +79,4 @@ solon-mobile.code-workspace
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 
 # Secrets
-android/app/google-services.json
\ No newline at end of file
+android/app/google-services.json
```

Comment:
need to add api token to this and once we completely phase out firebase, we can remove google-services